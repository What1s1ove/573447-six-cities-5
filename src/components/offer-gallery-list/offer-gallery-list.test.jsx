import * as React from 'react';
import {render} from 'enzyme';
import OfferGalleryList from './offer-gallery-list';

describe(`OfferGalleryList component`, () => {
  it(`should render OfferGalleryList component`, () => {
    const component = render(
        <OfferGalleryList
          imgPaths={[
            `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/5.jpg`,
          ]}
        />
    );

    expect(component).toMatchSnapshot();
  });
});
