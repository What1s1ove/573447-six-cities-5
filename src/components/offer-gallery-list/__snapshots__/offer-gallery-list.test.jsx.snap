// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferGalleryList component should render OfferGalleryList component 1`] = `
<div
  class="property__gallery-container container"
>
  <div
    class="property__gallery"
  >
    <div
      class="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        class="property__image"
        src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/5.jpg"
      />
    </div>
  </div>
</div>
`;
